app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+    @comments = @user.comments
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find_by_id(params[:id])
+    @comments = @post.comments
```

Comment:
- [ ] Use `last_five_comments` method to render the post's 5 most recent comments.